request.py

```diff
@@ -0,0 +1,127 @@
+import logging
+from datetime import datetime
+from sqlalchemy import exc
+import sqlalchemy
+from flask import jsonify, request, make_response, g
+from init import app
+import function
+# import utils
```

Comment:
Unused imports should be removed. You need to get rid of this reflex of commenting everything instead of just removing.